### DISCUSSION TITLE
JK DZ11 active balancer.

### DISCUSSION THREAD

**Discussion by RichGorrod** [syssi/esphome-jk-bms#516] [General]
Hello,

I new to this but does anyone have any info on getting Data from a JK DZ11 battery balancer
![Balancer](https://github.com/syssi/esphome-jk-bms/assets/141816621/f5a3cde4-44c0-450e-9b51-66fa1ae7b60c)

Can get in connect with Bluetooth but would like to get the info in Node red via RS485.

Have put one into a Seplos DIY. Just trying to get all cell votages into node red.

Many thanks.,

Cheers

Rich

**Comment by syssi** [comment]
Some details from another discussion: https://github.com/Louisvdw/dbus-serialbattery/discussions/1090

**Reply by syssi** [reply]
Some other issues & discussions:

https://github.com/syssi/esphome-jk-bms/issues/16
https://github.com/syssi/esphome-jk-bms/issues/17
https://github.com/syssi/esphome-jk-bms/pull/18/files

Long story short: We should poke around to get an idea which protocol is used to get an idea this device is supported out-of-the-box or not.

**Reply by syssi** [reply]
If you could do some testing we could try to move forward here. :-)

**Reply by syssi** [reply]
The device is supported now. See https://github.com/syssi/esphome-jk-bms/pull/18
